packages/lisk-transaction-pool/test/queue.ts

```diff
@@ -1,22 +1,148 @@
-/*
- * Copyright © 2018 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- *
- */
+import { Queue } from '../src/queue';
+import { Transaction } from '../src/transaction_pool';
+import { expect } from 'chai';
+import {transferTransactionInstances } from 'test_data';
+
 
 describe('Queue', () => {
-	describe('enqueueOne', () => {});
-	describe('enqueueMany', () => {});
-	describe('exists', () => {});
-	describe('removeFor', () => {});
-	describe('dequeueUntil', () => {});
-});
+    let queue: Queue;
+
+    beforeEach(() => {
+        queue = new Queue();
+    });
+
+    describe('enqueueOne', () => {
+        it('should add transaction to the queue', () => {
+            const transaction = transferTransactionInstances[0];
+            queue.enqueueOne(transaction);
+            expect(queue.transactions).to.include(transaction);
+        });
+
+        it('should add transaction to the queue index', () => {
+            const transaction = transferTransactionInstances[0];
+            queue.enqueueOne(transaction);
+            expect(queue.index[transaction.id]).to.deep.eq(transaction);
```

Comment:
let's use `deep.equal`. or simply `equal` here
Im not sure `mocha` has `eq` =0

Reply:
It passes the linting. and the tests pass, so there is `deep.eq` and i'd used it before. But I'll change it.